docs/source/usage/sync.rst

```diff
@@ -0,0 +1,12 @@
+sync
+======
+
+.. code-block:: bash
+
+    Sync will make sure that your current directory and dependencies are on the same branch.
```

Comment:
This should actually go outside of the `.. code-block:: bash` section. That should be reserved to show the output on the command line when you run `GitDepend help sync`